package.json

```diff
@@ -70,6 +70,7 @@
         "colors": "^1.1.2",
         "coveralls": "^3.0.0",
         "cross-env": "5.1.1",
+        "decentjs-lib": "git+ssh://git@github.com/DECENTfoundation/decentjs-lib",
```

Comment:
why not npm dependence?

Reply:
Because our decentjs-lib is not published on npm. There is decentjs-lib on npm published by somebody else.